python/cuml/linear_model/linear_regression.pyx

```diff
@@ -270,6 +314,66 @@ class LinearRegression(Base,
                           "column currently.", UserWarning)
             self.algo = 0
 
+        if 1 < y_cols:
+            # In the cuml C++ layer, there is no support yet for multi-target
```

Comment:
If possible, please move this branch to a separate function to improve modularity.